tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -75,3 +75,15 @@ Feature: get users
       | root@@@id         | %space_id%                       |
       | root@@@webDavUrl  | %base_url%/dav/spaces/%space_id% |
       | webUrl            | %base_url%/f/%space_id%          |
+
+
+  Scenario: admin user gets the group information of a user
```

Comment:
@ScharfViktor can we also add a scenario like `non-admin user gets the group information of another user` if feasible.

Reply:
Yes, true. Thanks)